src/proc/colorizer.cpp

```diff
@@ -136,7 +136,11 @@ namespace librealsense
 
     colorizer::colorizer()
         : colorizer("Depth Visualization")
-    {}
+    {
+        std::string device_name = this->get_info(RS2_CAMERA_INFO_NAME);
+        librealsense::aus_system_timer_start("COLORIZER", device_name);
```

Comment:
Looks much better..
Why do we need both timer and counter here?
counter I understand, we count number of filter we use.
What is the role of timer on Ctor? what kind of information we get here?

Reply:
I thought then to count how many times the colorizer filter was initialized,
and also to measure the time that this filter was working.. did not found other way to measure that.. (maybe I'm wrong?)
Anyway, now while we are discussing this, I think we should also count the number of total filters that are in use..

Reply:
Lets add this topic to the TODO list ticket.
No meaning of measure the time since object creation, we want the process time..

Reply:
removed